Blog - Churn vs Retention rate / A big guide to churn (#5222)
PostHog/posthog.com: contents/blog/customer-churn-analysis-guide.md

> 🚨 **Important:** Do not confuse monthly and annual churn rates. Monthly churn compounds into annual churn – i.e. a monthly churn rate of 5% equals a massive 46% annual churn rate. You can't just multiply your monthly churn rate by 12 to get your annual number, either. See [how to convert monthly churn to annual](#converting-monthly-churn-into-annual-churn-and-vice-versa) in the appendix.

### Why should you care about churn?

Reducing churn makes intuitive sense, but it's worth considering the practical impact of high churn and reducing it. To simplify somewhat, you can boil it down to three points:

  1. High churn is a likely sign you don't have product-market fit.

  2. High churn leads to increased sales and/or marketing spend to compensate.

  3. Reducing churn compounds – i.e. revenue **and** marketing ROI increase.

All are compelling reasons to reduce churn, but you need some context before you do. Is your current churn rate actually bad, or does it just **feel** bad? Let's dig into that next.

> ### What is a good churn rate?

Author: Going to re-org this section based on feedback tomorrow

## How to improve retention and reduce churn

So, how do you improve retention and reduce churn? Talking to users is a vital component – we're [constantly talking to users](/blog/making-something-people-want) – but here are some tactical moves to consider:

- **Check for involuntary churn:** Involuntary churn typically means billing problems – e.g. expired cards, out-of-date info etc. In e-commerce, abandoned baskets are a form involuntary churn.

- **Target different users:** Sometimes poor retention comes from acquiring the wrong users. This could be a marketing problem, or you might have the wrong [ideal customer profile](/blog/creating-ideal-customer-profile) for your product.

- **Learn about your power users:** It's great to have power users, but they're also more sensitive to product changes. It's important to identify who they are, what they do, and what they value. They're your biggest advocates, so keep them happy. See: [How to identify and analyze power users in PostHog](/tutorials/power-users).

> - **Improve your product:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. I'm told this is "creating value" in the official business lingo.

Author: ```suggestion
- **Ship new features:** If you're talking to your users, and learning about your power users, then you'll be well-placed to build awesome new features your existing (and potential) customers will love. Shipping new features, or improving current ones, means users get more value from your product.
```

- **Focus on customer success:** Customer success > outbound sales a lot of the time. [Our customer success team](/handbook/small-teams/customer-success) owns all inbound contact requests, assisting new users and introducing them to the right PostHog people at the right time. If you can't afford a dedicated team, give the job to someone until you do. PostHog CEO, James Hawkins, owned this area in our early days.

> - **Re-engage your users:** Users who like your product can sometimes be distracted by other things. Well-timed re-engagement messages, such as showcasing an underused featured, can be extremely powerful. [Path analysis](/product/user-paths) is a useful tool for understanding where users get distracted.

Author: ```suggestion
- **Re-engage users:** Users who like your product can get distracted by other things. Well-timed re-engagement messages, such as showcasing an underused featured, can be extremely powerful. [Path analysis](/product/user-paths) is a useful tool for understanding where users get distracted.
```

### The things you need to improve retention

You're gonna need a few things to track, analyze and reduce customer churn, such as:

1. The ability to identify your new/current users (e.g. job role, location etc.)

1. An analytics tool to help you track user churn, and analyze use behavior.

1. Session recording, so you can watch real interactions with your product.

1. A way to test new features and improvements to validate if they work.

1. The ability to safely roll out new features to test groups and specific cohorts

1. Product-minded engineers or product managers who talk to users and ship fast

Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.

And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.

Check out [our product page](/product) if this sounds interesting to you.

/pitch over

> ## Takeaways

Author: Will probably add to / change some stuff here too.

Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.

To conclude, here are {CHANGE THIS} takeaways you can use to sound smart in meetings, or perhaps share on LinkedIn 😱 with a hashtag – I won't judge, not publicly at least:

> 1. Early-stage startups shouldn't get too hung up on how high or low churn is, but you should care **which direction** it's headed. The actual rate of churn / retention is more important once you have product-market fit and care about revenue.

Reviewer: Should they care about this? There is an argument that they shouldn't care which direction churn is moving in. If they are growing fast, and churn is growing slower, that can still be a good situation.

Author: I might caveat this slightly, but I think it's a matter of degrees. Very slow increase in churn might be fine, but a faster rate would be bad and wipe out that growth quite quickly. It's more like "keep an eye on it and don't let it go crazy".

> 1. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than one with an ARPC of $1,000 per month.

Reviewer: ```suggestion
1. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than an enterprise SaaS with an ARPC of $10,000 per month.
```

1. PostHog is awesome and you should totally adopt it at your startup. "Nobody got fired for buying PostHog" said someone, probably me.

- TODO: WILL PROBABLY ADD ANOTHER TWO HERE

### Further reading

If you're interested in early-stage startups and product-led growth, you may find these articles useful: 

- PostHog CEO, James Hawkins, on [how we found our Ideal Customer Profile](/blog/creating-ideal-customer-profile)

- My guide to [building an AARRR pirate funnel](/blog/aarrr-pirate-funnel) for tracking your most important metrics

- Another piece from James on [how we made something people want](/blog/making-something-people-want)

> You can also sign up to our newsletter over there 👉. We send it every two weeks. It's full of articles like this.

Reviewer: ```suggestion
You can also sign up to our newsletter over there 👉 (or 👇 on mobile). We send it every two weeks. It's full of articles like the one you just read.
```

Author: Hmm, good spot. Might be easier to turn this into a link.

In other words, if you have 1,000 customers and 100 of them cancel their subscription, your monthly churn rate would be 10% and your retention rate would be 90%. That's it.

Typically, "churn" and "customer churn" are used interchangeably, but there are numerous formulas for revenue churn/retention metrics – e.g. net revenue retention, gross revenue churn etc. I won't be covering them here, but [ADD LINK TO A GOOD SOURCE ON THE TOPIC].

Those are the basics, but there's more worth learning. Let's start with churn rate.

## Understanding churn rate

Here's the simplest formula for churn rate:

> **Customer churn rate** = (Churned customers during a specific period / Total customers at the start of that period) x 100

Where:

> **Churned customers** = (Customers at the start of the period - Customers at the end of the period) + New customers acquired in that period

The majority of companies should track churn monthly and/or annually. Those interested in tracking churn daily or weekly are better off using a [cohort retention table](#how-to-read-and-use-a-cohort-retention-table). More on those later.

> > 🚨 **Important:** Do not confuse monthly and annual churn rates. Monthly churn compounds into annual churn – i.e. a monthly churn rate of 5% equals a massive 46% annual churn rate. You can't just multiply your monthly churn rate by 12 to get your annual number, either. See [how to convert monthly churn to annual](#converting-monthly-churn-into-annual-churn-and-vice-versa) in the appendix.

Author: ```suggestion
🚨 **Important:** Do not confuse monthly and annual churn rates. Monthly churn compounds into annual churn – i.e. a monthly churn rate of 5% equals a massive 46% annual churn rate. You can't just multiply your monthly churn rate by 12 to get your annual number, either. See [how to convert monthly churn to annual](#converting-monthly-churn-into-annual-churn-and-vice-versa) in the appendix.
```

- **Mid-Market B2B SaaS:** Anything up to 5% monthly churn is OK for a B2B SaaS that **doesn't** sell to big enterprises, but something around 3% is preferable if your average revenue per customer (ARPC) is above $250 per month. Anything below 2% is excellent. A monthly churn rate of 3% is equivalent to 30.6% annually – i.e. a company with 1,000 customers would lose 306 over 12 months.

- **Early-stage B2B SaaS:** A monthly churn rate above 5% is to be expected for any early-stage company B2B SaaS, so don't worry too much if you're growing fast and you're above that benchmark. Just monitor your trend and dig deeper if you don't see it decline as you mature. A monthly churn of 5.65% is equivalent to 50.24% annually – i.e. a company with 1,000 companies would lose 502 over 12 months, more than half.

> - **Enterprise B2B SaaS:** Products selling to large enterprises (think more than 1,000 employees) have lower churn, mainly because large companies change slowly and will generally bill annually. A monthly churn rate up to 2% is good. Anything less than 1% is excellent. A monthly churn rate of 1% is equivalent to 11.3% annually – i.e. a company with 1,000 customers will lose 114 over 12 months.

Author: ```suggestion
- **Enterprise B2B SaaS:** Products selling to large enterprises (think more than 1,000 employees) have lower churn, mainly because large companies change slowly and are billed annually. A monthly churn rate up to 2% is good. Anything less than 1% is excellent. A monthly churn rate of 1% is equivalent to 11.3% annually – i.e. a company with 1,000 customers will lose 114 over 12 months.
```

- **Check for involuntary churn:** Involuntary churn typically means billing problems – e.g. expired cards, out-of-date info etc. In e-commerce, abandoned baskets are a form involuntary churn.

- **Target different users:** Sometimes poor retention comes from acquiring the wrong users. This could be a marketing problem, or you might have the wrong [ideal customer profile](/blog/creating-ideal-customer-profile) for your product.

- **Learn about your power users:** It's great to have power users, but they're also more sensitive to product changes. It's important to identify who they are, what they do, and what they value. They're your biggest advocates, so keep them happy. See: [How to identify and analyze power users in PostHog](/tutorials/power-users).

- **Ship useful new features:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. If you solve more problems for your customers, they're more likely to retain.

> - **Improve speed/reliability/ease of use:** Users may love your product, but find it slow, buggy or hard to use. Again, talking to users will make it obvious if you have a problem here.- **Move customers to annual contracts:** Remember how companies selling to enterprises have lower churn? It's not an accident. Consider offering discounts or incentives to existing customers to move onto annual contracts. Once they do, they're less likely to churn.

Author: ```suggestion
- **Improve speed/reliability/ease of use:** Users may love your product, but find it slow, buggy or hard to use. Again, talking to users will make it obvious if you have a problem here.

- **Move customers to annual contracts:** Remember how companies selling to enterprises have lower churn? It's not an accident. Consider offering discounts or incentives to existing customers to move onto annual contracts. Once they do, they're less likely to churn.
```

1. The ability to identify your new/current users (e.g. job role, location etc.)

1. An analytics tool to help you track user churn, and analyze use behavior.

1. Session recording, so you can watch real interactions with your product.

1. A way to test new features and improvements to validate if they work.

1. The ability to safely roll out new features to test groups and specific cohorts

1. Product-minded engineers or product managers who talk to users and ship fast

Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.

And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.

Check out [our product page](/product) if this sounds interesting to you.

/pitch over

## Takeaways

Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.

> To conclude, here are {CHANGE THIS} takeaways on all things churn analysis:

Author: ```suggestion
To conclude, here are five takeaways on all things churn analysis:
```

1. Higher churn in an early-stage startup is fine, so long as you're growing fast and finding impactful ways to improve your product. Declining churn is a good sign you either have, or are approaching, product-market fit.

1. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than an enterprise SaaS with an ARPC of $10,000 per month.

> 1. Use cohort retention tables to understand what types of users, or features, positively or negatively impact churn. This will help guide product development, marketing, and areas to experiment in.

Author: ```suggestion
1. Use cohort retention tables to understand what types of users, or features, positively or negatively impact churn. This will help guide product development and marketing.
```

---
date: 2023-02-07
title: "Retention vs Churn Rate: A complete guide to churn analysis"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - andy-vandervell
featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
featuredImageType: full
category: Product growth
tags:
  - Product analytics
  - Product metrics
  - Guides
  - Explainers
---

You know what's cooler than gaining new customers? Retaining them.

In this guide to customer churn analysis, I'm exploring churn rate and retention rate:
- How are they different?
- How do you measure them? 
- Which one would win a fight?<sup>*</sup>

> These and other useful questions (like [how to reduce churn](#how-to-improve-retention-and-reduce-churn) will be answered. Let's get to it.

Reviewer 2: I'd just include 'how to improve them' as the fourth bullet and cut this down

> <sup>(*strictly speaking, they'd cancel each other out... read the next section to understand why)</sup>

Reviewer 2: ```suggestion
```

## What's the difference between retention rate and churn rate?

Not much, really: **churn rate** is the percentage of customers who **stop using** your product during a given period of time; **retention rate** is the percentage of **existing customers** who **continue using** your product. 

In other words, if you have 1,000 customers and 100 of them cancel their subscription, your monthly churn rate would be 10% and your retention rate would be 90%. That's it.

> Simple, right? Yes, but these definitions ignore three important points:

Reviewer 2: ```suggestion
```

> 1. There are numerous ways to measure and analyze them

Reviewer 2: ```suggestion
```

> 1. Just knowing what they are won't help you improve them

Reviewer 2: ```suggestion
```

> 1. While both are important, in my view **retention rate** is more useful (most of the time)

Reviewer 2: ```suggestion
```

> I'll explore these points further as we go. See [Takeaways on churn analysis](#takeaways-on-churn-analysis) for a TL;DR.

Reviewer 2: ```suggestion
```

## Understanding churn rate

Here's the simplest formula for churn rate:

> > **Customer churn rate** = (Churned customers during a specific period / Total customers at the start of that period) x 100

Reviewer: ```suggestion
> **Customer churn rate** = Churned customers during a specific period / Total customers at the start of that period x 100
```
Based on order of operations, you don't need the brackets here.

Author: I think I prefer to keep them in. Makes it a little easier parse.

> Where churn customers equal:

Reviewer 2: ```suggestion
Where:
```

> > **Churned customers** = (Customers at the start of the period - Customers at the end of the period) + New customers acquired in that period

Reviewer: ```suggestion
> **Churned customers** = Customers at the start of the period - Customers at the end of the period + New customers acquired in that period
```

The majority of companies should track churn monthly and/or annually. Those interested in tracking churn daily or weekly are better off using a [cohort retention table](#how-to-read-and-use-a-cohort-retention-table). More on those later.

> 🚨 **Important:** Do not confuse monthly and annual churn rates. Monthly churn compounds into annual churn – i.e. a monthly churn rate of 5% equals a massive 46% annual churn rate. You can't just multiply your monthly churn rate by 12 to get your annual number, either. See [how to convert monthly churn to annual](#how-to-convert-monthly-churn-into-annual-rate) in the appendix.

### What is a good churn rate?

> Benchmarking churn rate is tricky because:

Reviewer 2: I hate it when I read an article and the answer to a question I have is 'it's complicated' followed by a bunch of caveats. I think this section would be stronger if it went something like: 

It depends:
- For consumer apps, e.g. social media, it's X
- For B2B software with small contract sizes (e.g. $10-100/month) it's Y
- etc.

It sounds like you have the data, so why not surface a bit more of it, vs. having people run off to another article

Reviewer 2: e.g. this is way better https://www.lennysnewsletter.com/p/what-is-good-retention-issue-29

1. As I just mentioned, monthly churn rates compound into annual churn. It's **really important** to know which number you're talking about!

2. Companies selling to enterprises have lower churn than those selling to SMBs on cheaper, monthly contracts. They will also report churn annually – see point #1.

3. Studies investigating churn rates by industry use different methodologies and data sets. They often come to conflicting conclusions.

4. Companies who self report churn rate are liable to, erm... lie. After all, who wants to admit they have a churn problem? They might also define churn in their own, special way.

5. What's "good" for a mature business is very different to what's good for a pre-product-market fit, early-stage startup.

With all that said, and having read countless studies, meta-analyses, and finger-in-the-air opinions on the topic, here are my conclusions:

- **Monthly churn of 0.5% or less** is outstanding for any company, but an unrealistic expectation for B2C products. It's equivalent to an annual churn rate of 5.84% – i.e. a company with 1,000 customers would lose 58 over 12 months.

> - **Monthly churn of 1.5 to 2%** is still very good, especially for mid-market B2B companies that don't sell to enterprises. It's equivalent to an annual churn rate of 16.59% – i.e. a company with 1,000 customers would lose 166 over 12 months.

Reviewer: What happened to 0.5-1.5?

- **Monthly churn above 5%** is to be expected for any early-stage company B2B SaaS, and a good benchmark for B2C subscription products. When you're early-stage, hitting an absolute number is less important than seeing your churn rate improving.

See [more on churn rate benchmarks](#more-on-churn-rate-benchmarks) in the appendix for some of the sources I used to reach these conclusions.

### Problems with churn rate and alternative methods

Other articles you read about churn rate may introduce alternative ways to measure it lifted directly from [this 2011 article](https://shopify.engineering/defining-churn-rate-no-really-this-actually-requires-an-entire-blog-post) by a data scientist at Shopify.

It's a dense affair that highlights some interesting quirks about churn rate, which I'll quickly summarize:

- Extreme growth can lead to distorted churn figures – such as reporting lower churn when user behavior hasn't actually changed. High-growth startups should be wary as a result.

- Because churn rate measures churn among **existing customers**, it can hide churn problems among new, incoming customers.

> The Shopify article explores several complex alternative churn formulas to address these problems. It's a laudable objective, but in my view **totally unnecessary**.

Reviewer 2: Not sure you have made an actual argument here that the Shopify article is wrong? I'd consider skipping the whole 'problems with alternative methods' section as you don't really go deep enough here - to do this properly, you'd have to explain each alternate method, then rebut it, which takes us into academic paper territory...

Reviewer: Agree, if someone is at this point I think they are bought into churn rate as a metric. I'd never heard of that Shopify article and it is 12 years old. I think this section could derail a lot of readers.

Is churn rate a perfect metric? No. Do perfect metrics exist? Also, no. Our objective here isn't mathematical perfection, it's a comparable figure that's easy to understand. The more complicated a metric becomes, the less likely people will use it or understand it. 

Bottom line: You should **never** rely on just one number to measure success. See also why the whole "one number that matters" is a trap.

> 💡**PostHog Tip:** A funnel analysis tracking signup through activation, and conversion-to-paid is a better way to investigate churn among new users. Read our guide to [building funnels in PostHog](/tutorials/funnels) for more on this.

## Understanding retention rate

So that's churn rate, but what about retention rate and why, as I claimed earlier, is it a more useful metric? (most of the time) 

To calculate your overall customer retention over a period you need to know:

- How many customers you have at the start of a period (A)
- How many customers you have at the end of a period (B)
- How many new customers were added during that period. (C)

You can then calculate your retention as a percentage (R) using the following formula:

> > ((B - C) / A) * 100 = R

Reviewer 2: I preferred your churn formula where you used the names of the terms, instead of 'A, B, C' as it's easier to follow

> > ((B - C) / A) * 100 = R

Reviewer: ```suggestion
> (B - C) / A * 100 = R
```

For example, if you have 100 customers at the start of a year, gain 35 new customers during the year and end the year with a total of 80 customers then your retention rate would be:

> > ((80-35)/100)*100 = 45% retention

Reviewer: ```suggestion
> (80-35)/100*100 = 45% retention
```

Meaning that each year a little more than half of your customers are churning and your usage as a whole is in decline. This would be a worrying sign!

As you'll know if you didn't skip the intro, retention is just the inverse of churn rate. So, the being the case, why is retention rate **more useful** than churn rate?

Because you can analyze retention using a cohort retention table like this:

![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)

> I'll explain how to read retention tables in a moment, but here's the rub... retention tables allow you to answer questions like:

Reviewer 2: I would expect an explanation on how to read this table immediately below it - maybe put these bullets above the table?

- Which features do users come back to over and over?
- Which features do users try and never use again?
- What actions contribute to users retaining or churning?
- What properties do retaining users have that or churning users lack?

If you can answer these questions, you're on the way the improving retention and reducing churn. And that's why retention rate is more useful, most of the time.

### How to read (and use) a cohort retention table

> Trust me, it's less complicated than it looks.

Reviewer 2: ```suggestion
```

![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)

Above is a PostHog retention table for an imaginary Dropbox clone called Hedgebox. In this retention table we're looking at weekly repeat usage,  configured as:

> `Unique users` who `signed up for the first time` in the last 8 weeks who then came back and triggered the `interacted with file` event in PostHog.

To read it, just start on the left and work your way across:

- **Cohort** Denotes the week these users signed up – i.e. the cohort these users belong to.

- **Size:** Shows the size of that cohort – a useful reference to avoid comparing a very small cohort to a much larger one.

- **Week 0:** Gives you the percentage of users in the cohort who used Hedgebox in that week. As it's week 0, this number will always be 100%.

> - **Week 1-8:** Shows the percentage of users in that cohort who returned to the product in each subsequent week – white boxes indicate periods that 'still in progress', blue boxes are completed periods.

Reviewer 2: To clarify - the percentage is of the previous box, not the original box on the far left, correct?

Reviewer 2: Or is it the other way round - it's worth over-explaining this!

Author: > To clarify - the percentage is of the previous box, not the original box on the far left, correct?

It's the percentage of the original box on the far left. I'm considering whether visualizing this might be better.

In this example, we can see retention settles at ~20% from 'week 4' onwards in most cohorts. 

> That's a useful baseline, but what happens if we filter by just people who use Microsoft Windows?

Reviewer: ```suggestion
That's a useful baseline, but what happens if we filter by just people who use Windows?
```

![new user retention windows](../images/blog/retention-vs-churn/new-user-retention-windows-users.png)

It's obvious Windows users retain worse than the baseline, dipping as low as 15% in week 4 onwards in the 'Dec 11' cohort. 

Do macOS users retain better than Windows users? Let's check...

![new user retention mac os](../images/blog/retention-vs-churn/retention-table-macos.png)

They do! While the exact values vary, macOS users consistently retain better than Windows users. 

> This prompts several avenues os investigation. Is the Windows Hedgebox experience worse and, if so why? Does Hedgebox solve some fundamental problem for macOS users that makes them retain better? Are Windows users demographically different to macOS users?

Reviewer: ```suggestion
This prompts several avenues of investigation:
- Is the Windows Hedgebox experience worse and, if so why? 
- Does Hedgebox solve some fundamental problem for macOS users that makes them retain better? 
- Are Windows users demographically different to macOS users?
```

This is just one example using demo data in PostHog. You could also filter users by:

- Their age
- The browser they use

> - The email account they used to sign up 

Reviewer: No sure email is relevant for cohort retention

- The country or city they live in
- The number of files they upload
- The types of files they upload
- User job titles, company type or myriad other properties

Cohort retention tables are also useful for tracking revenue retention, especially if you charge based on usage. Positive revenue retention (i.e. more than 100%) indicates users increase their spend over time. Needless to say, this is a good thing.

> > 😱 **Pedantic Sidebar:** For those thinking "if retention and churn rate are just the inverse of each other, why can't I create a churn rate table instead?" My answer is this... yes, you could, but it would be a dumb thing to do. Why? Because retention tables make more intuitive sense. Be a rebel if you like, but you're dead to me. 

Reviewer 2: I don't get this argument. It feels like another 'because I said so.' Why would it be dumb? You've just explained what a retention table is to me, so I (the casual reader) have no idea what a churn table looks like.

## How to improve retention and reduce churn

> "Let's reduce churn" isn't quite akin to "nobody ever got fired for buying IBM", but it's worth remembering why reducing churn is desirable:

Reviewer 2: This is another section where the title is 'how to ___ ' but then you digress with a little bit of context first. I'd almost have a separate section at the top that is 'why reducing churn is important' aka 'why should you care about this article'

Author: Moving this into the 'Understanding churn rate' section

1. High churn is a likely sign you don't have product-market fit.

2. High churn leads to increased sales and/or marketing spend to compensate.

3. Reducing churn compounds – i.e. revenue **and** marketing ROI increase.

> So, how do you improve retention and reduce churn? The glib answer is "make product better", but we can do a bit better than that:

Reviewer 2: This section generally isn't super convincing/actionable - maybe you could pull the top 5 actually tactical useful things from here: https://www.lennysnewsletter.com/p/how-to-increase-your-products-retention

- **Check for involuntary churn:** Involuntary churn typically means billing problems – e.g. expired cards, out-of-date info etc. In e-commerce, abandoned baskets could also be an involuntary churn problem.

> - **Improve new user activation:** New user activation is an [important B2B product metric](/blog/b2b-saas-product-metrics). Use funnel analysis and session recordings to identify problems in your onboarding flow. Sometimes small tweaks to user flows can have a big impact.

Reviewer 2: A failure to activate shouldn't be consider churn IMO (and most people who write about this stuff that I've read take that stance)

- **Talk to users:** Stop me if you've heard this before, but talking to users is vital in any context – not just when you're trying to solve a retention problem. We're constantly talking to users to understand what they need, and problems we can solve. We built the [PostHog User Interview](https://github.com/posthog/user-interview-app) and [Feedback apps](https://github.com/PostHog/feedback-app) to help us do se. We document our [user feedback process](/handbook/product/user-feedback) in our public handbook.

- **Consider marketing and product alignment:** Sometimes poor retention comes from acquiring the wrong users. This could be a marketing problem, or you might have the wrong ideal customer profile.

- **Learn about your power users:** It's great to have power users, but they're also more sensitive to product changes. It's important to identify who they are, what they do, and what they value so you can avoid retention problems. See: [How to identify and analyze power users in PostHog](/tutorials/power-users).

> - **Build great new features:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. I'm told this is "creating value" in the official business lingo.

Reviewer 2: "The glib answer is "make product better", but we can do a bit better than that:"

*proceeds to be glib*

> - **Create a customer success team:** Customer success > outbound sales a lot of the time. [Our customer success team](/handbook/small-teams/customer-success) owns all inbound contact requests, assisting new users and introducing them to the right PostHog people at the right time. An effective CS team will help onboard new customers and retain them long term.   

Reviewer 2: Alternatively (/more realistically for startups) - make customer success someone's explicit responsibility. At PostHog, that is what our CEO James did.

> - **Buy or copy the new hotness:** Aka, doing a Zuckerberg. This probably isn't a viable option for most companies, but YOLO right? (Yes, I did include this one mostly for the lols. It's not serious advice, probably.)

Reviewer 2: ```suggestion
```

> ### The things you need to improve retention

Reviewer: Could you combine this section with the past section? Add each bullet as a sentence to the bullet in this last section.

Author: Possibly. I'll have a look at this once I'm done fixing up some other bits.

> You're gonna need a few things to track, analyze and improve customer churn, such as:

Reviewer: You say improve retention in the title then improve churn in the first sentence, maybe both?

1. An analytics tool to help you track user churn, and analyze use behavior.

1. Session recording so you can watch real interactions with your product.

1. A way to test new features and improvements to validate if they work.

1. The ability to safely roll out new features to test groups and specific cohorts

1. Product-minded engineers or product managers who talk to users and ship fast

Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.

And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.

Check out [our product page](/product) if this sounds interesting to you.

/pitch over

## Takeaways on churn analysis

Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.

> To conclude, here are five takeaways you can use to sound smart in meetings, or perhaps share on LinkedIn 😱 with a hashtag – I won't judge, not publicly at least:

Reviewer 2: Maybe too far, but at least if you take out some of the earlier snark, this will be funny

> 1. Generally speaking, **churn rate is more useful to exec teams**, investors, and finance peeps, and **retention is more useful for product teams**. That's not to say one group shouldn't also care about the other, but it's useful context for when to use them.

Reviewer 2: This isn't a summary - this is totally new information! I'd also argue it's wrong - at any board meeting I've been at, we always talk about retention first and foremost. In particular net dollar retention.

2. Early-stage startups shouldn't get too hung up on how high or low churn is, but you should care **which direction** it's headed. The actual rate of churn / retention is more important once you have product-market fit and care about revenue.

3. Don't overcomplicate how you measure churn. It's easier to just understand when it can lead you astray and factor that in. No metric is perfect.

4. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than one with an ARPC of $1,000 per month.

> 5. PostHog is awesome and you should totally adopt it at your startup. "Nobody got fired for buying PostHog" said someone, probably me.

Reviewer 2: Ok now this is funny
